arlbench/autorl/autorl_env.py

```diff
@@ -162,18 +164,47 @@ def _get_objectives(self) -> list[Objective]:
             raise ValueError("Please select at least one optimization objective.")
 
         objectives = []
+        objective_arguments = []
         cfg_objectives = list(set(self._config["objectives"]))
         for o in cfg_objectives:
-            if o not in OBJECTIVES:
+            if o not in OBJECTIVES and ("_".join(o.split("_")[:1]) in OBJECTIVES or "_".join(o.split("_")[:2]) in OBJECTIVES or "_".join(o.split("_")[:3]) in OBJECTIVES):
```

Comment:
I think we should do this more generally than checking for the three different cases explicitly.